packages/react-core/src/components/Switch/examples/Switch.md

```diff
@@ -171,8 +171,12 @@ import React from 'react';
 import { Switch } from '@patternfly/react-core';
 
 <React.Fragment>
-  <Switch id="uncontrolled-switch-on" label="Message when on" labelOff="Message when off" defaultChecked={false} />
-  <br />
-  <Switch id="uncontrolled-no-label-switch-on" aria-label="Message when on" defaultChecked={true} />
+  <Switch
+    id="uncontrolled-switch-on"
+    aria-label="Uncontrolled switch"
+    label="Message when on"
+    labelOff="Message when off"
+    defaultChecked={false}
+  />
```

Comment:
Similar to @nicolethoen comment from above. For now I could see either just removing the `aria-label` or showing two separate switches (one using the `aria-label` without a visible text label, and one using the `label`/`labelOff` props).

Updating the label/aria-label for uncontrolled might be something to revisit in the future.

Reply:
I guess I'm a little unclear about this and @nicolethoen comments. I added the label/labelOff because this works properly for rendering on the screen.

The issue is that with uncontrolled, the aria-labelledby doesn't switch with it so might not make sense (a switch to "on" from "off" would still point to label, not labelOff). Adding a generic aria-label that indicates the checkbox and doesn't point to a specific label seemed like a solution for this... screen readers will still note the checkbox change I believe?

It seems like we'd either have to do what I did, remove the label/labelOff and use aria-label, or remove the labelOff. What would be preferred?

Fine but no visible label:

```
  <Switch
    id="uncontrolled-switch-no-label"
    aria-label="Uncontrolled switch"
    defaultChecked={false}
  />
```

Odd because though the label/labelOff will display correctly with the change, the labelledby will always point to label (which has something such as "on")

```
  <Switch
    id="uncontrolled-switch-labels"
    label="Message when on"
    labelOff="Message when off"
    defaultChecked={false}
  />
```

Will switch between the labels, but may not matter as the labels will have the same text content

```
  <Switch
    id="uncontrolled-switch-labels"
    label="An uncontrolled switch"
    defaultChecked={false}
  />
```

Reply:
Ah good catch. Based on your latest comment, I would personally say one of the following:

1. Include both your first and third example in your comment above (one using only `aria-label` and one only using `label`), just to show the uncontrolled variant with a visible and non-visible label
2. Only include your third example that uses only the `label` prop